src/gui/editors/PianoRoll.cpp

```diff
@@ -65,6 +67,7 @@
 #define MiddleButton MidButton
 #endif
 
+using namespace std;
```

Comment:
Why did you add `using namespace std;` into this file, while other files don't have this?

Reply:
I needed this to drag `std::move` into the global scope. The other files\* only use `make_unique`, which is dragged into scope by stdshims.h.

\* except for ComboBoxModel.cpp; I just pushed a commit that has it also `using namespace std;` instead of explicitly prefixing all the std types with `std::`.

Also, I avoid `using namespace` directives in any _headers_, as that's usually bad practice (e.g. https://stackoverflow.com/a/14575955/216292 ).

If preferred, I can explicitly specify `using std::move; using std::unique_ptr;` etc, instead of dragging in the whole namespace - just let me know.

Reply:
I prefer `using std::move;`, `using std::unique_ptr;`, etc. However, this would be fine if it doesn't bring any side effects.

Reply:
Okay, I switched to explicitly writing `using std::move;`, `using std::unique_ptr;`, etc.